### ISSUE TITLE
Create a quickstart

### ISSUE THREAD

**Issue by jacobtomlinson** [opsdroid/opsdroid#1484]
@jacobtomlinson I initially found the overview quite confusing, do you think the docs would benefit from a "Quickstart," or something along those lines?

_Originally posted by @Gideon357 in https://github.com/opsdroid/opsdroid/issues/1198#issuecomment-630492674_

**Comment by jacobtomlinson** [comment]
I would support extending the quickstart on the home page of the docs to be a little more comprehensive. Currently it is just install instructions. 

https://docs.opsdroid.dev/en/stable/

Perhaps it should show configuring the slack connector and adding a simple local skill.

**Comment by Gideon357** [comment]
@jacobtomlinson good idea w/ the slack connector, my shell version was very basic. I just wanted to limit the need for API keys and set, but does slack need that?

**Comment by jacobtomlinson** [comment]
I think your idea to use the shell connector is good. But nobody will actually use the shell connector once they are done playing, so we should link to more docs on configuring "real" connectors.
